mssqlcli/mssqlclioptionsparser.py

```diff
@@ -0,0 +1,159 @@
+        dest=u'row_limit',
+        default=os.environ.get(MSSQL_CLI_ROW_LIMIT, None),
+        metavar=u'',
+        help=u'Set threshold for row limit prompt. Use 0 to disable prompt.')
+
+    args_parser.add_argument(
+        u'--less-chatty',
+        dest=u'less_chatty',
+        action=u'store_true',
+        default=False,
+        help=u'Skip intro on startup and goodbye on exit.')
+
+    args_parser.add_argument(
+        u'--auto-vertical-output',
+        dest=u'auto_vertical_output',
+        action=u'store_true',
+        default=False,
+        help=u'Automatically switch to vertical output mode if the result is wider than the terminal width.')
+
+    args_parser.add_argument(
+        u'-N', u'--encrypt',
+        dest=u'encrypt',
+        action=u'store_true',
+        default=False,
+        help=u'SQL Server uses SSL encryption for all data if the server has a certificate installed.')
+
+    args_parser.add_argument(
+        u'-C', u'--trust-server-certificate',
+        dest=u'trust_server_certificate',
+        action=u'store_true',
+        default=False,
+        help=u'The channel will be encrypted while bypassing walking the certificate chain to validate trust.')
+
+    args_parser.add_argument(
+        u'-l', u'--connect-timeout',
+        dest=u'connection_timeout',
+        default=0,
+        metavar=u'',
+        help=u'Time in seconds to wait for a connection to the server before terminating request.')
+
+    args_parser.add_argument(
+        u'-K', u'--application-intent',
+        dest=u'application_intent',
+        metavar=u'',
+        help=u'Declares the application workload type when connecting to a database in a SQL Server Availability Group.')
+
+    args_parser.add_argument(
+        u'-M', u'--multi-subnet-failover',
+        dest=u'multi_subnet_failover',
+        action=u'store_true',
+        default=False,
+        help=u'If application is connecting to AlwaysOn AG on different subnets, setting this provides faster '
+             u'detection and connection to currently active server.')
+
+    args_parser.add_argument(
+        u'-a', u'--packet-size',
+        dest=u'packet_size',
+        default=0,
+        metavar=u'',
+        help=u'Size in bytes of the network packets used to communicate with SQL Server.')
+
+    args_parser.add_argument(
+        u'-A', u'--dac-connection',
+        dest=u'dac_connection',
+        action=u'store_true',
+        default=False,
+        help=u'Connect to SQL Server using the dedicated administrator connection.')
+
+    args_parser.add_argument(
+        u'--enable-sqltoolsservice-logging',
+        dest=u'enable_sqltoolsservice_logging',
+        action=u'store_true',
+        default=False,
+        help=u'Enables diagnostic logging for the SqlToolsService.')
+
+    args_parser.add_argument(
+        u'--prompt',
+        dest=u'prompt',
+        metavar=u'',
+        help=u'Prompt format (Default: \\d> ')
+
+    return args_parser
+
+
+def get_parser():
```

Comment:
get_parser implies we are returning an existing instance. Since were creating a new one each call, I'd prefer we name this create_parser or build_parser.